searchlib/src/vespa/searchlib/docstore/writeablefilechunk.cpp

```diff
@@ -225,12 +233,18 @@ WriteableFileChunk::read(LidInfoWithLidV::const_iterator begin, size_t count, IB
                         assert(chunk == _active->getId());
                         buffer = _active->getLid(li.getLid());
                     }
-                    visitor.visit(li.getLid(), buffer);
+                    auto copy = std::make_unique<char[]>(buffer.size());
```

Comment:
Should we consider using `Alloc` here in case of some crazy big documents?

Reply:
Good point, fixed.